tools/bouncer_worker/src/tasks/importToy.js

```diff
@@ -0,0 +1,291 @@
+/**
+ *  Copyright (C) 2017 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>
+ */
+
+const { MongoClient, GridFSBucket } = require('mongodb');
+const fs = require('fs');
+const { config } = require('../lib/config');
+const runCommand = require('../lib/runCommand');
+const logger = require('../lib/logger');
+
+const logLabel = { label: 'TOY' };
+
+const accumulateCollectionFiles = (modelDir, modelId) => {
+	const importCollectionFiles = {};
+
+	fs.readdirSync(modelDir).forEach((file) => {
+		// remove '.json' in string
+		let collectionName = file.split('.');
```

Comment:
Would it not be better to do `.replace(/\.json$/,"")` instead? At the moment, popping will always remove the last element even if it isn't `json`.